Content.Shared/_Sunrise/AddWantedStatus/AddWantedStatusComponent.cs

```diff
@@ -0,0 +1,16 @@
+using Content.Shared.Actions.Components;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared._Sunrise.AddWantedStatus;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class AddWantedStatusComponent : Component
+{
+    [DataField]
+    public EntProtoId<TargetActionComponent> Action = "ActionAddWanted";
+
+    [DataField, AutoNetworkedField]
+    public EntityUid? ActionEntity;
```

Comment:
```suggestion
    [DataField, ViewVaribales]
    public EntProtoId<TargetActionComponent> Action = "ActionAddWanted";

    [ViewVaribales, AutoNetworkedField]
    public EntityUid? ActionEntity;
```

Юид не должен быть датафилдом + ВВ